modules/special-rules/pages/rapid-fire.adoc

```diff
@@ -2,12 +2,17 @@
 
 `Unit ability`
 
-{blank}[TODO: add introductory text.]
+These units specialise in laying down heavy fire as soon as they take up position.
+
+[NOTE.e40k]
+====
+In comparison to _Epic 40,000_, we've changed the {rapid-fire} ability from doubling the Firepower value to increasing it by 1. This doesn't affect any of the original units with {rapid-fire} as they all had Firepower 1, but this allows us to add new units with Firepower 2 or higher and {rapid-fire}, without them becoming overpowered.
```

Comment:
```suggestion
In {e40k}, when the _Rapid Fire_ ability is active it *doubles* the unit's Firepower value.
We've changed this so that the effect scales in linear fashion when we add {rapid-fire} to units that have Firepower `2` or more.
This doesn't affect any of the original {e40k} units that have _Rapid Fire_, as they all have Firepower `1` -- so the net result remains the same.
```